src/app/submission/sections/form/section-form.component.spec.ts

```diff
@@ -318,6 +322,77 @@ describe('SubmissionSectionFormComponent test suite', () => {
       expect(comp.hasMetadataEnrichment(newSectionData)).toBeFalsy();
     });
 
+    describe('inCurrentSubmissionScope', () => {
+      beforeEach(() => {
+        // @ts-ignore
+        comp.formConfig = {
+          rows: [
+            {
+              fields: [
+                {
+                  selectableMetadata: [{ metadata: 'scoped.workflow' }],
+                  scope: 'WORKFLOW',
+                } as FormFieldModel
+              ]
+            },
+            {
+              fields: [
+                {
+                  selectableMetadata: [{ metadata: 'scoped.workspace' }],
+                  scope: 'WORKSPACE',
+                } as FormFieldModel
+              ]
+            },
+            {
+              fields: [
+                {
+                  selectableMetadata: [{ metadata: 'dc.title' }],
+                } as FormFieldModel
+              ]
+            }
+          ]
+        };
+      });
+
+      describe('in workspace scope', () => {
+        beforeEach(() => {
+          // @ts-ignore
+          comp.submissionObject = { type: WorkspaceItem.type };
+        });
+
+        it('should return true for unscoped fields', () => {
+          expect((comp as any).inCurrentSubmissionScope('dc.title')).toBe(true);
+        });
+
+        it('should return true for fields scoped to workflow', () => {
+          expect((comp as any).inCurrentSubmissionScope('scoped.workspace')).toBe(true);
```

Comment:
Tiny thing...this test should say `it('should return true for fields scoped to workspace'...`.  It wrongly says "workflow"